tools/controlcenter.py

```diff
@@ -0,0 +1,24 @@
+# This file is part of the EESSI build-and-deploy bot,
+# see https://github.com/EESSI/eessi-bot-software-layer
+#
+# The bot helps with requests to add software installations to the
+# EESSI software layer, see https://github.com/EESSI/software-layer
+#
+# author: Kenneth Hoste (@boegel)
+# author: Thomas Roeblitz (@trz42)
+#
+# license: GPLv2
+#
+
+
+def control_center_process_comment(request_body):
```

Comment:
Maybe a shorter function name like `cc_process_comment` is better?

Reply:
We'll need a separate function for new comments vs updated comments, so perhaps `cc_process_new_comment` and `cc_process_updated_comment`?